postgres/datadog_checks/postgres/statements.py

```diff
@@ -181,6 +194,35 @@ def _get_pg_stat_statements_columns(self):
                 self._stat_column_cache = col_names
                 return col_names
 
+    def _check_called_queries(self):
+        pgss_view_without_query_text = self._config.pg_stat_statements_view
+        if pgss_view_without_query_text == "pg_stat_statements":
+            pgss_view_without_query_text = "pg_stat_statements(false)"
+
+        with self._check._get_main_db() as conn:
+            with conn.cursor(cursor_factory=CommenterCursor) as cursor:
+                called_queryids = []
+                query = QUERY_COUNTS_QUERY.format(pg_stat_statements_view=pgss_view_without_query_text)
+                rows = self._execute_query(cursor, query, params=(self._config.dbname,))
+                for row in rows:
+                    queryid = row[0]
+                    if queryid is None:
+                        continue
+
+                    calls = row[1]
+                    calls_changed = self._query_count_cache.set_calls(queryid, calls)
+                    if calls_changed:
+                        called_queryids.append(queryid)
+
+                self._check.gauge(
+                    "postgresql.pg_stat_statements.called",
```

Comment:
this metric name is a bit confusing and prefix `postgresql` makes it a postgres metric. if it's for debug purpose, let's prefix it `dd.postgres`. maybe
```suggestion
                    "dd.postgres.pg_stat_statements.calls_changed",
```